index.js

```diff
@@ -1 +1,78 @@
-const app = "I don't do much."
+var cake = {
```

Comment:
Thanks!. Can never have enough practice getting into existing code to fix bugs or add features.